sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,22 +1,15 @@
 # Release History
 
-## 1.8.0-beta.2 (Unreleased)
+## 1.8.0 (2023-02-03)
 
 ### Features Added
+- Added support to configure `clientOptions`, `httpLogOptions`, `retryPolicy`, `retryOptions` and `addPolicy` on Identity credentials.
```

Comment:
I believe we discussed calling out in changelogs which features are being GA'ed from beta series.
adding a subheader something along the lines of "Features being GA'ed from v1.8.0-beta.1"